WheelWizard/Features/Settings/ADDING_SETTINGS.md

```
@@ -35,6 +36,14 @@ MY_DOLPHIN_SETTING = RegisterDolphin(
 );
 ```
 
+### Recomp
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)